dojo/tools/snyk_code/parser.py

```diff
@@ -0,0 +1,262 @@
+import json
+
+from cvss.cvss3 import CVSS3
+from dojo.models import Finding
+
+
+class SnykCodeParser(object):
+    def get_scan_types(self):
+        return ["Snyk Code Scan"]
```

Comment:
if you implemented this parser to distinguish between Snyk and Snyk Code, I would rather recommend to either do it like it was done with RustyHog: https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/tools/rusty_hog/parser.py or with Checkmarx: https://github.com/DefectDojo/django-DefectDojo/blob/c5d896b2a606985f41669388962e47409f4866ba/dojo/tools/checkmarx/parser.py#L16